src/generators/generateActionStubs.ts

```diff
@@ -51,15 +51,24 @@ export class GenerateActionStubs {
     if (!response?.assistants) {
       return;
     }
-
-    for (const sig of response?.assistants) {
-      let allParamsSorted = this.getAllParamsSorted(sig, stubs);
-      pythonContent += `@assistant('${sig.id}')\n`;
-      pythonContent += this.buildAssistantParmDec(sig, allParamsSorted);
-      pythonContent += `async def assistant_${snakeCase(sig.name)}(\n`;
-      pythonContent += this.buildAssistantParams(sig, allParamsSorted);
-      pythonContent += `) -> None:\n`;
-      pythonContent += `    pass\n\n\n`;
+    const labs = await client.queryLabs();
+    if (!labs) {
+      return;
+    }
+    for (const lab of labs.labs) {
+      pythonContent += `class ${lab.name.charAt(0).toUpperCase() + camelCase(lab.name).slice(1)}Assistants:\n`;
+      for (const sig of response.assistants) {
+        if (lab.id === sig.constraint.labId) {
+          let allParamsSorted = this.getAllParamsSorted(sig, stubs);
+          pythonContent += `\t@staticmethod\n`;
+          pythonContent += `\t@assistant('${sig.id}')\n`;
+          pythonContent += this.buildAssistantParmDec(sig, allParamsSorted);
+          pythonContent += `\tasync def assistant_${snakeCase(sig.name)}(\n`;
```

Comment:
Since we're putting assistants into a class whose name contains "Assistants," can we remove "assistant" from the method name?

Reply:
no, unless we also do what JamesB said and restrict characters...or more specifically, dont allow an assistant to be named 111BestAssistant.  Since python doesnt like that.